### ISSUE TITLE
Trimmed Playback With Xlib Crashes On Exit

### ISSUE THREAD

**Issue by ghost** [LunarG/gfxreconstruct#628]
Trimmed captures that do not include the application's cleanup code can leave the application in an invalid state.
This can result in a crash on exit when using Xlib.
Playback should ensure that Xlib objects are destroyed properly.

This can be replicated on Ubuntu 20.04 with vkcube built with Xlib support...

From your Vulkan SDK install directory:
```
cd <vulkan/sdk/path>/1.2.198/source/Vulkan-Tools/
mkdir build
cd build/
cmake -DBUILD_WSI_XCB_SUPPORT=OFF -DBUILD_WSI_XLIB_SUPPORT=ON -DBUILD_WSI_WAYLAND_SUPPORT=OFF -DBUILD_WSI_DIRECTFB_SUPPORT=OFF -DCUBE_WSI_SELECTION=XLIB ..
make
```

...binaries can be found in `<vulkan/sdk/path>/1.2.198/source/Vulkan-Tools/build/cube/`

**Comment by panos-lunarg** [comment]
Hi @ghost
Can you give this another go with #690 in place. I'm suspecting your issue has been fixed.

**Comment by panos-lunarg** [comment]
Closing this as fixed by #690
